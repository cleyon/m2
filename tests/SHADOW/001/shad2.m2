@define __INPUT__ Foo
@for I 1 2
@define __INPUT__ Bar
@next I
Afterward, __INPUT__ is @__INPUT__@
