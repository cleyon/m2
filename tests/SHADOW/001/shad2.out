Afterward, __INPUT__ is Bar
